README.md

```diff
@@ -1,60 +1,49 @@
-# generator-react-zeal
-
-Yeoman generator for Zeal's React boilerplate
-
-Expected to be used in the root of a Rails or Phoenix project configured with
-the Zeal client app strategy.
-
+### Yeoman Generator for [Zeal's](https://codingzeal.com/) React Boilerplate
 [![npm version](https://img.shields.io/npm/v/generator-react-zeal.svg)](https://www.npmjs.com/package/generator-react-zeal)
 [![CircleCI](https://circleci.com/gh/CodingZeal/generator-react-zeal.svg?style=shield&circle-token=62c69a799a2c2a4ad70148f0a4a398b38279ad91)](https://circleci.com/gh/CodingZeal/generator-react-zeal)
 
+This generator can be used with `yarn` or `npm`. `generator-react-zeal` is a [yeoman](http://yeoman.io/) generator and follows the usage pattern commonly found with the yo cli.
 
-## Installation
+***
 
+#### Yarn
 ```
-# Install yo cli, as well as this generator, globally using either yarn or npm
-
-# with yarn
+# Install yo cli, as well as this generator
 yarn global add yo generator-react-zeal
 
-# with npm
-npm install -g yo generator-react-zeal
-```
+# If necessary, create new directory and `cd` into it
+mkdir myApp && cd myApp
 
-## Usage
-`generator-react-zeal` is a [yeoman](http://yeoman.io/) generator and follows the usage pattern commonly found with the yo cli.
+# Run the generator
+yo react-zeal
+
+# Install
+yarn
 
+# start your app on http://localhost:3000
+yarn start
+```
+#### Npm
 ```
-# Create new directory and `cd` into it
+# Install npm cli, as well as this generator
+npm install -g yo generator-react-zeal
+
+# If necessary, create new directory and `cd` into it
 mkdir myApp && cd myApp
 
 # Run the generator
 yo react-zeal
 
-# install all the things
-
-# with yarn
-yarn
-
-# with npm
+# Install
 npm install
 
 # start your app on http://localhost:3000
-
-# with yarn
-yarn start
-
-# with npm
 npm start
 ```
 
-Voila! Now craft with React.
+***
 
-## Example
-
-Checkout our usage in the context of Phoenix app - https://github.com/CodingZeal/phoenix-react-apollo-demo
-
-## Context
+#### Context
```

Comment:
I'm not sure about all of the h4's in here now.  Looking at the rendered document, I find it a bit less scannable with the smaller headings.